src/tasks/deploy.mk

```diff
@@ -85,6 +85,20 @@ deplo%: ## deploy: deploy the app to heroku
 	heroku pipelines:promote -a $(HEROKU_APP_STAGING)
 	heroku dyno:scale web=0 -a $(HEROKU_APP_STAGING)
 
+	curl \
+  		--header "Content-Type: application/json" \
+  		--header "x-api-key: $(CHANGE_API_KEY)" \
```

Comment:
Is this a new key that went into vault? If so, does that mean we need to add it to each project in their shared folder?

Reply:
Yep, I will be adding that to vault 👍 - though I was under the impression it was going to be one shared folder? Might be getting confused about that.

Reply:
The secret value goes into `teams/next/shared/continuous-integration`, but for each project, you also need to add the secret key to `teams/next/[project-name]/shared`

Reply:
Weird quirk of next-vault-sync, but for `teams/next/shared/continuous-integration` there's no need to opt-in. Every CircleCI project will get those environment variables.

Reply:
Oh, even better then :)

Reply:
>Weird quirk of next-vault-sync, but for teams/next/shared/continuous-integration there's no need to opt-in. Every CircleCI project will get those environment variables.


I'm a fan